src/veritysetup/veritysetup-generator.c

```diff
@@ -220,21 +240,214 @@ static int determine_devices(void) {
         return 1;
 }
 
+static int create_disk(
+                const char *name,
+                const char *data_device,
+                const char *hash_device,
+                const char *roothash,
+                const char *options,
+                const char *source) {
+
+        _cleanup_free_ char *n = NULL, *du = NULL, *hu = NULL, *e = NULL, *filtered = NULL, *du_escaped = NULL, *hu_escaped = NULL, *name_escaped = NULL;
```

Comment:
nit: please break this line

Reply:
How long is a max line?

Reply:
When it feels too long in a diff :-)

```
- Don't break code lines too eagerly. We do **not** force line breaks at 80ch,
  all of today's screens should be much larger than that. But then again, don't
  overdo it, ~109ch should be enough really.
```

https://github.com/systemd/systemd/blob/master/docs/CODING_STYLE.md

Reply:
thanks